src/core/Module.ts

```diff
@@ -142,12 +143,7 @@ export class Module {
     try {
       const withSourcemaps = this.isSourceMapRequired();
       if (!this.isJavascriptModule()) {
-        this.ast = buntis.parseTSModule(this.contents, {
-          directives: true,
-          jsx: this.props.extension === '.tsx',
-          next: true,
-          loc: withSourcemaps,
-        }) as ASTNode;
+        this.ast = parseTypeScript(this.contents, { jsx: this.props.extension === '.tsx', locations: withSourcemaps });
```

Comment:
what happens when people type jsx in ts files?

Reply:
it's not supported by typescript ;-)